lib/ansible/modules/cloud/amazon/batch_compute_environment.py

```diff
@@ -0,0 +1,514 @@
+#!/usr/bin/python
+# (c) 2017, Jon Meran <jonathan.meran@sonos.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
Just a docs review so you pass shippable till I can check out and your changes and look at it in-depth.... Applies to the other two files too.

Up here you'll want the metadata:
ANSIBLE_METADATA = {'metadata_version': '1.0',
                                         'status': ['preview'],
                                         'supported_by': 'community'}